pynn_spinnaker/simulator.py

```diff
@@ -112,6 +112,14 @@ def _wait_for_transition(self, placements, allocations,
             raise Exception("Unexpected core failures "
                             "before reaching %s state (%u/%u)." % (to_state, cores_in_to_state, num_verts))
 
+    def _configure_software_watchdog(self):
+        # If software watchdog is disabled, write zero to machine's
+        # SV struct, otherwise, write default from sv struct file
+        # **NOTE** x, y and p should already be in context
+        w = (0 if self.disable_software_watchdog
+             else self.machine_controller.structs["sv"]["soft_wdog"].default)
+        self.machine_controller.write_struct_field("sv", "soft_wdog", w)
```

Comment:
I'd probably just ignore the structure default and set to true or false based on the config option you've made.